src/Sarif/Autogenerated/FileRole.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CodeDom.Compiler;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Values specifying the roles played by the file in the analysis.
+    /// </summary>
+    [Flags]
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.49.0.0")]
+    public enum FileRole
+    {
+        Default,
```

Comment:
There is no "default" role. Modify the code hint to name the "0" enum value "`None`". I know there's a knob for that.

Reply:
Ah, there it is, it's `"zeroValueName"`.

Reply:
Fixed